templates/web/fixmystreet.com/static/posters.html

```diff
@@ -45,7 +55,7 @@ <h2 id="activist-pack">The FMS Activist Pack</h2>
     <li>Ideas for letting more people know about FixMyStreet</li>
 </ul>
 
-<a href="/cobrands/fixmystreet/posters/activist-pack.zip" class="download-button">Download ZIP file</a>
+<a href="/cobrands/fixmystreet/posters/activist-pack.zip" onclick="trackGoodieDownload('activist-pack', '/cobrands/fixmystreet/posters/activist-pack.zip'); return false;" class="download-button">Download ZIP file</a>
```

Comment:
Could you just use `$(this).attr('href')` inside `trackGoodieDownload()` to save writing the URL out twice?